keyboards/40percentclub/mf68/config.h

```diff
@@ -20,8 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xCEEB
-#define PRODUCT_ID      0x0510
+#define VENDOR_ID       0x3430 // "40"
```

Comment:
This makes me very uncomfortable, as I know newer VID assignments are very close to this range.  

```
echo -n "40percentclub" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
```
This produces 0xA825, which is a much safer value. 

Also, why not stay with CEEB?

Reply:
Because the other 40percentclub board in VIA , the nein, has it like that